### PR TITLE
cloud: update billing details page

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add **Summary by service** and **Usage details**.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Review by langyuezhang — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/tidb-cloud-billing.md [hunk 1]
```diff
@@ -56,6 +56,15 @@ To view the billing details, perform the following steps:
 2. Click **Billing**.
 3. Click **Bills**. The billing details page is displayed.
 
+The billing details page shows the billing summary by project and by service. You can also see the usage details and download the data in CSV format.
+
+> **Note:**
+>
+> The total amount in the monthly bill may differ from that in the daily usage details due to differences in precision:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:63]:
```suggestion
> The total amount in the monthly bill might differ from that in the daily usage details due to differences in precision:
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-billing.md:63]: https://developers.google.cn/style/word-list#may

**Review by langyuezhang — APPROVED** [review]

**Comment by langyuezhang** [comment]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**